repack-postgres.sh

```diff
@@ -7,7 +7,9 @@ set -eu
 # https://www.enterprisedb.com/products-services-training/pgbindownload
 
 VERSION=9.6.3-1
+PPC64LE_VERSION=9.6.18
 BASEURL="https://get.enterprisedb.com/postgresql"
+PPC64LE_BASEURL="https://github.com/amitsadaphule/postgres-ppc64le-bin-factory/releases/download/$PPC64LE_VERSION"
```

Comment:
Is there some other source to get the postgres binaries?  I don't think it's a good idea to rely on anyone's personal github.

Reply:
I had tried to use the rpm available for centos 7 earlier. But that has a dependency on advanced-toolchain-10 and also throws errors related to libpq. Other than that, I couldn't really find a binary distribution tarball for postgres-9.x for ppc64le. So, I setup a travis job to build postgres and upload the tarball to github releases.

Reply:
I'll merge this as is, and if needed, we can create a prestodb project for this.

Reply:
Thanks @rschlussel!